modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/thunks/fieldChange.es.js

```diff
@@ -117,13 +121,20 @@ export default function fieldChange({
 						if (fieldInstance.isDisposed()) {
 							return;
 						}
+						const mergedPages = mergePages(
```

Comment:
@rijema, please add a break line here. Sorry for not let you know about this on my previous review. Usually, we separate variable declarations from statements, like `if`.